tracks/rust/exercises/gigasecond/mentoring.md

```diff
@@ -0,0 +1,73 @@
+### Concepts
+
+- Integer
+- constant
+- struct (We're returning the `Utc` struct, which is the field in the
```

Comment:
You might want to consider linking `struct`, `Utc` and `DateTime` to the official docs/reference and `Chrono crate` to the chrono crate page.

Reply:
Great thought! Done :)